Makefile

```diff
@@ -111,7 +113,7 @@ unit-docker: metering-src-docker-build
 		$(METERING_SRC_IMAGE_REPO):$(METERING_SRC_IMAGE_TAG) \
 		make unit
 
-integration: bin/test2json
+integration: $(TEST2JSON_BIN_OUT) $(DEPLOY_METERING_BIN_OUT)
```

Comment:
Nit: these aren't necessary in this specific PR since your not using the new binary in e2e/integration yet.